### ISSUE TITLE
Expansion short cut

### ISSUE THREAD

**Issue by dlfivefifty** [JuliaApproximation/ContinuumArrays.jl#131]
A common pattern to expand a function is:
```julia
P = Legendre()
x = axes(P,1)
f = P / P \ exp.(x)
```
But perhaps a shortcut would be useful so this is one line:
```julia
expand(Legendre(), exp)
```
That is, add the function:
```julia
function expand(P, f)
    x = axes(P,1)
    P / P \ f.(x)
end
```

**Comment by TSGut** [comment]
Yes, this is a good idea. I think for people who may really only want to do very specific computations with the packages (sort of like what I've seen some people do with ApproxFun), having syntax that can side-step the quasimatrix concept would be helpful.

It may even be worth having an optional integer argument that gives you only the first $N$ entries of the vector (without computing the whole thing).
